modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ApexClientCodegen.java

```diff
@@ -161,8 +161,10 @@ public void processOpts() {
 
     @Override
     public void preprocessOpenAPI(OpenAPI openAPI) {
-        Info info = openAPI.getInfo();
-        String calloutLabel = info.getTitle();
+        String calloutLabel = openAPI.getInfo().getTitle();
+        if (namedCredential != null && !namedCredential.trim().isEmpty()) {
```

Comment:
What about using `StringUtils.isNotEmpty` instead?

Reply:
@wing328 Very good idea. I'm not a Java developer by trade, so I'm a bit gun shy with libraries.